applications/nublado/values.yaml

```diff
@@ -128,10 +154,16 @@ controller:
           cpu: 4.0
           memory: 12Gi
 
-      # -- Volumes that should be mounted in lab pods. Currently this only
-      # supports NFS volumes and must specify `containerPath`, `server`,
-      # `serverPath`, and `mode` where mode is one of `ro` or `rw`.
-      volumes: {}
+      # -- Volumes that should be mounted in lab pods. This supports NFS,
+      # HostPath, and PVC volume types (differentiated in source.type)
+      # volumes:
+      # - containerPath: "/project"
+      #   mode: "rw"
+      #   source:
+      #     type: nfs
+      #     serverPath: "/share1/project"
+      #     server: "10.87.86.26"
```

Comment:
If you really want to include an example, it has to go after the setting, not before, or it gets sucked into the Markdown documentation and then badly misformated and makes it hard to read.